samples/js/browser/index.html

```diff
@@ -583,8 +583,8 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
                 // This can happen for two broad classes or reasons.
```

Comment:
Maybe we should mention that Reason is used to differentiate whether it is Error or EndOfStream